src/ServiceControl.Audit.Persistence.RavenDb5/Indexes/AuditCountIndex.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ServiceControl.Audit.Persistence.RavenDb.Indexes
+{
+    using System.Linq;
+    using Raven.Client.Documents.Indexes;
+    using ServiceControl.Audit.Auditing;
+    using ServiceControl.Audit.Auditing.MessagesView;
+    using ServiceControl.Audit.Monitoring;
+
+    public class AuditCountIndex : AbstractIndexCreationTask<ProcessedMessage, DailyAuditCount>
```

Comment:
By the way, do y'all think it's OK here to use the DailyAuditCount type, which lives in the base persistence project, as the ReduceResult here? I wrestled with the coupling implications. Alternative would be to use an internal class `AuditCountIndex.Result` but then the structure would mean a LOT of mapping.

Meanwhile I think the complex structure of the result is good, I'm trying to keep the index so that it's completely time-based on the day so that when audit expiry happens and all the audit information for a day evaporates, it's just 1 thing that Raven has to also throw away for this index.

Reply:
I don't see any issue directly using the `DailyAuditCount` type. It's already a DTO on its own, no need for more mappings.